src/react/ui-elements/ReauthDialog.jsx

```diff
@@ -17,26 +15,16 @@ const ReauthDialog = () => {
   const errorMessage = useSelector((state: AppState) =>
     state.uiErrors.errorType === 'byAuth' ? state.uiErrors.errorMsg : null,
   );
-  const pathname = useSelector(
-    (state: AppState) => state.router.location.pathname,
-  );
   const oidcLogout = useSelector((state: AppState) => state.auth.oidcLogout);
 
-  const dispatch = useDispatch();
-
-  const reauth = pathName => {
-    dispatch(networkAuthReset());
-    dispatch(loadClients()).then(() => dispatch(push(pathName)));
-  };
-
   if (!needReauth) {
     return null;
   }
 
   return (
     <Modal
       id="reauth-dialog-modal"
-      close={() => reauth(pathname)}
+      close={() => window.location.reload()}
```

Comment:
```suggestion
      close={window.location.reload}
```

Reply:
I will keep it this way because it will cause a :
```
Uncaught TypeError: Illegal invocation react
```